stack.yaml

```diff
@@ -16,11 +16,27 @@ packages:
 # These entries can reference officially published versions as well as
 # forks / in-progress versions pinned to a git hash. For example:
 extra-deps:
+  - aeson-2.2.3.0
+  - binary-0.8.9.2
+  - bytestring-0.12.1.0
+  - character-ps-0.1
+  - containers-0.7
+  - directory-1.3.8.5
+  - extra-1.8
+  - filepath-1.4.300.2
   - lsp-2.7.0.0
   - lsp-types-2.3.0.0
+  - parsec-3.1.17.0
+  - process-1.6.20.0
   - set-extra-1.4.2
+  - text-2.1.1
+  - time-1.9.3
+  - unix-2.8.5.1
+  - Win32-2.14.1.0
   - git: https://git.ps.informatik.uni-kiel.de/curry/curry-frontend.git
-    commit: bd1750a68e011e56c176491a558885b5268173b5
+    commit: 67adff16d1166286db21e2fbddb88724674c95e9
+
+allow-newer: true
```

Comment:
Unfortunately we need quite a few `extra-deps`, since the Curry frontend decided to pull these in on top of the Stackage snapshot. `allow-newer` is needed due to conflicts, e.g. the newest version of `lsp` still requires `extra` 1.7, while the Curry frontend requires 1.8.

This list should get much shorter again once these packages make it to the next Stackage snapshot.